packages/theming/demo/themeProvider.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, ArgsTable, Canvas, Story, Markdown } from '@storybook/addon-docs';
+import { ThemeProvider, DEFAULT_THEME, PALETTE } from '@zendeskgarden/react-theming';
+import { PaletteStory } from './stories/PaletteStory';
+import README from '../README.md';
+
+<Meta
+  title="Packages/Theming/ThemeProvider"
+  component={ThemeProvider}
+  subcomponents={{ DEFAULT_THEME, PALETTE }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+## ThemeProvider
+
+<Canvas>
+  <Story name="ThemeProvider" args={{ theme: DEFAULT_THEME }}>
+    {args => <ThemeProvider {...args} />}
```

Comment:
I realize this is currently empty – the `Story` was needed to get generated API docs. We can fill it with something meaningful in the future if/when 💡 strikes